### DISCUSSION TITLE
Are there plans to support default Android JS Engine (not Hermes)?

### DISCUSSION THREAD

**Discussion by dimaportenko** [software-mansion/react-native-reanimated#1877] [General]
I have a project with dependencies which not support Hermes (an app is crashing). I would like to use reanimated 2 so I'm curious if there any plans to support the default Android JS engine?

**Comment by jgonet** [comment]
Yup, there's an ongoing effort to support JSC as well. See #1842 for more details. This PR should be included in the next minor version of Reanimated.

**Reply by dimaportenko** [reply]
This is great! Thanks for the feedback.

**Reply by shamilovtim** [reply]
Would appreciate v8 support as well!

**Reply by hanhailong** [reply]
Strong support for V8
